test/unit/annotation_spec.js

```diff
@@ -1540,6 +1742,43 @@ describe('annotation', function() {
         done();
       }, done.fail);
     });
+
+    it('should correctly inherit Contents from group-master annotation ' +
+       'if parent has ReplyType == Group', function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      annotationDict.set('Contents', 'Correct Text');
```

Comment:
Just like in the tests above, we should check all fields instead of only the contents, i.e., contents, title, modification date and color (no need for the flags).